operator/test/test.sh

```diff
@@ -79,15 +79,18 @@ init() {
 setup_test() {
   echo "[Setup]"
   echo -n "  - Namespace configuration: "
-  if ! kubectl get namespace $NAMESPACE >/dev/null 2>&1; then
+  if kubectl get namespace $NAMESPACE >/dev/null 2>&1; then
+    # Delete old pipelines resources
+    kubectl delete -n "$NAMESPACE" pipelineruns,pipelines,tasks --all
+  else
     echo -n "."
     kubectl create namespace "$NAMESPACE" >/dev/null
+    # Wait for pipelines to set up all the components
+    while [ "$(kubectl get serviceaccounts -n "$NAMESPACE" | grep -cE "^pipeline ")" != "1" ]; do
+      echo -n "."
+      sleep 2
+    done
   fi
-  # Wait for pipelines to set up all the components
```

Comment:
I would not move that while block. The fact that the namespace already exists is not a guarantee that the SA has been setup.

Reply:
Fixed